### PR TITLE
Niyaz dev12/issue5

### LINKED ISSUE #5: Create the footer component.
**Is your feature request related to a problem? Please describe.**

**Describe how the solution should behave**
The design should be exactly the same thing. For icons use SVGs and the routes are already built in the Navbar component.

**Screenshots of the design/if any**
image

**Additional context**

### PR DESCRIPTION
**Pull request by Niyaz-Dev12**
Responsive Footer created with placeholder data for social links and contact-us info
Closes #5

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                <p className="font-bold text-lg md:self-center lg:self-start">
+                  Contact Us
+                </p>
+                <p>Email Address: example@example.com</p>
+                <p>Phone Number: 0770 000 00 00</p>
+              </div>
+            </div>
+            <div className="flex flex-col">
+              <div className="flex justify-center mt-2">
+                <Link to={HOME_ROUTE} className="mx-5">
```
Comment (RawandDev) [inline src/components/Footer.jsx]: Please make these look like a link in the UI. Because people wouldn't know this is a link unless they hover over it.

**Review by RawandDev — CHANGES_REQUESTED** [review]

**Comment by RawandDev** [comment]
I see the whole logo is an svg. You could download only the book as svg then write Libvery. It would be easier to align I guess since the alignment of Libvery and Contact us is not the same
Untitled
.

File: src/components/Footer.jsx (hunk 1 above)
Comment (Niyaz-Dev12) [reply src/components/Footer.jsx]: i added the hover color effect, as for making it obvious it's a link underline may work but i haven't seen footers with that

**Review by MuhannedNoman — APPROVED** [review]
Hey @Niyaz-Dev12 nice work here LGTM. Feel free to merge.

**Review by RawandDev — APPROVED** [review]
LGTM

**Pull request merged**